### DISCUSSION TITLE
2020 Proxy values due to lack of sampling due to COVID

### DISCUSSION THREAD

**Discussion by brianlangseth-NOAA** [pfmc-assessments/california-data#9] [General]
Thank you Melanie Parker for providing these. Values for 2025 cycle species are in the "[recreational-fishery > proxy 2020 data](https://github.com/pfmc-assessments/california-data/tree/main/recreational-fishery/proxy%202020%20data)" folder alongside existing values for the 2023 species. Also provided is a readme ([READ_ME_CDFWRec_2020 AvgProxyValue Methods.xlsx](https://github.com/pfmc-assessments/california-data/blob/main/recreational-fishery/proxy%202020%20data/READ_ME_CDFWRec_2020%20AvgProxyValue%20Methods.xlsx)) that describes how these values were estimated.

**Comment by MParkerCDFW** [comment]
1. To get full-year estimates, add the average proxy values to the estimates obtained from RecFIN. Average proxy values are not available by mode (PR/PC). If is it necessary to distinguish between PR/PC for the assessment, reasonable methods would be to use the ratio of PR/PC from the rest of 2020 and apply that to the avg proxy values, or the ratio of PR/PC from Apr-June from 2017-2019 (the data used to generate the avg proxy values) and apply that to the avg proxy values.
2. The reduction proportions to generate the average proxy values were based qualitative review of state and local shelter-in-place/stay-at-home, and closure restrictions from April-June 2020 to determine a reasonable reduction in effort and resulting catch. During April, more strict restrictions were in place, and the reductions to the 2017-2019 avgs are greater than for June, when restrictions were lifting and locations/businesses were opening back up. Additionally, fewer ocean access points and sport businesses are in more northerly ports, so reductions in the northern part of the state were greater than those in the southern parts of the state.
